### ISSUE TITLE
Outdated documentation

### ISSUE THREAD

**Issue by FedeG** [gaearon/react-hot-loader#648]
### Description
The migration-to-30 documentation (https://github.com/gaearon/react-hot-loader/tree/master/docs#migration-to-30) are outdated, all examples import HotContainer but is AppContainer.

### Solution
Change all imports to:
```javascript
import { AppContainer as HotContainer } from 'react-hot-loader'
// or
import { AppContainer } from 'react-hot-loader' // and replace HotContainer to AppContainer in the example implementation
```

**Comment by FedeG** [comment]
Related with https://github.com/gaearon/react-hot-loader/issues/647

**Comment by neoziro** [comment]
@FedeG yes you are right, I will try to clean documentation this week-end.

**Comment by neoziro** [comment]
Fixed in 00404dd0805260441339de4771816b1b87df118a
